assignment/poker/PokerEvaluator.md

```diff
@@ -0,0 +1,167 @@
+# Poker Hand Strength Evaluator
+
+## Task Description
+
+Your task is to develop an algorithm for comparing the strength of poker hands. 
+
+### Poker Games
+
+You should support one to three poker game types (the more the better):
+* Texas Hold'em - input is board of 5 cards, and multiple hands of 2 cards each
+
+  A value of a Texas Hold'em hand is the best possible value out of all possible subsets of 
+  5 cards from the 7 cards which are formed by 5 board cards and 2 hand cards.
+  See [Texas Hold'em rules](https://en.wikipedia.org/wiki/Texas_hold_%27em).
+
+* Omaha Hold'em - input is board of 5 cards, and multiple hands of 4 cards each
+
+  A value of an Omaha Hold'em hand is the best possible value out of all possible subsets of 
+  5 cards from the 7 cards which are formed by 3 out of 5 board cards and 2 out of 4 hand cards. 
+  See [Omaha Hold'em rules](https://en.wikipedia.org/wiki/Omaha_hold_%27em).
+
+* Five Card Draw - input is multiple hands of 5 cards each
```

Comment:
Should this go first, as the simplest one?

Reply:
I thought about it, but I thought then everyone will just do that and call it a day and it will effectively have simplified our task.